tutorials/wallet/blitz-wallet/tutorial.yml

```
@@ -0,0 +1,25 @@
+id: 794bdac4-1af4-49d5-9ea5-abb8228ca196
+
+level: beginner
+
+category: mobile
+
+professor_id: 7261131b-4ede-4d68-a299-5941584f5592
```

```suggestion
professor_id: 81a82a0f-3003-4459-9d6a-67f860efb20e
```
Not the right prof id